### PR TITLE
[6.3] Fix CLI ContentAccess applicable availability

### PR DESCRIPTION
**Pull request by ldjebran**
Use a concrete package to verify packages availability, the test was duplicating errata test availability but widely

```console
pytest -v tests/foreman/cli/test_contentaccess.py::ContentAccessTestCase
============================================ test session starts =============================================
platform linux2 -- Python 2.7.14, pytest-3.3.1, py-1.5.2, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/2.7.14/envs/sat-6.3/bin/python2.7
cachedir: .cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.20.1, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1, fauxfactory-1.1.1
collected 4 items                                                                                            
2018-01-09 18:09:37 - conftest - DEBUG - Collected 4 test cases


tests/foreman/cli/test_contentaccess.py::ContentAccessTestCase::test_negative_rct_not_shows_golden_ticket_enabled <- robottelo/decorators/__init__.py PASSED [ 25%]
tests/foreman/cli/test_contentaccess.py::ContentAccessTestCase::test_positive_erratum_installable <- robottelo/decorators/__init__.py PASSED [ 50%]
tests/foreman/cli/test_contentaccess.py::ContentAccessTestCase::test_positive_list_installable_updates <- robottelo/decorators/__init__.py PASSED [ 75%]
tests/foreman/cli/test_contentaccess.py::ContentAccessTestCase::test_positive_rct_shows_golden_ticket_enabled <- robottelo/decorators/__init__.py PASSED [100%]

============================================= 0 tests deselected =============================================
======================================== 4 passed in 1676.61 seconds =========================================
```

### CODE REVIEW

**Comment by ldjebran** [comment]
added UI fix
```console
pytest -v tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase
============================================ test session starts =============================================
platform linux2 -- Python 2.7.14, pytest-3.3.1, py-1.5.2, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/2.7.14/envs/sat-6.3/bin/python2.7
cachedir: .cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.20.1, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1, fauxfactory-1.1.1
collected 7 items                                                                                            
2018-01-09 22:07:26 - conftest - DEBUG - Collected 7 test cases


tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase::test_negative_list_available_packages <- robottelo/decorators/__init__.py SKIPPED [ 14%]
tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase::test_negative_visual_indicator_with_restricted_subscription <- robottelo/decorators/__init__.py PASSED [ 28%]
tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase::test_positive_list_available_packages <- robottelo/decorators/__init__.py PASSED [ 42%]
tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase::test_positive_list_installable_updates <- robottelo/decorators/__init__.py PASSED [ 57%]
tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase::test_positive_visual_indicator_on_activation_key_details <- robottelo/decorators/__init__.py PASSED [ 71%]
tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase::test_positive_visual_indicator_on_hosts_subscription <- robottelo/decorators/__init__.py PASSED [ 85%]
tests/foreman/ui/test_contentaccess.py::ContentAccessTestCase::test_positive_visual_indicator_on_manifest <- robottelo/decorators/__init__.py PASSED [100%]

============================================= 0 tests deselected =============================================
=================================== 6 passed, 1 skipped in 3140.95 seconds ===================================
```

File: tests/foreman/cli/test_contentaccess.py [hunk 1]
```diff
@@ -134,19 +136,21 @@ def setUpClass(cls):
     @run_only_on('sat')
     @tier2
     def test_positive_list_installable_updates(self):
-        """Run `hammer host errata list` and assert all updates are listed on
-        packages tab updates and not only those for attached subscriptions.
+        """Ensure packages applicability is functioning properly.
```
Comment (oshtaier) [inline tests/foreman/cli/test_contentaccess.py]: why does test case description is changed so much?

Comment (ldjebran) [reply tests/foreman/cli/test_contentaccess.py]: This test repeat the same doc as the next test, but it was intended to check package applicability

Comment (ldjebran) [reply tests/foreman/cli/test_contentaccess.py]: plus, this a pure CLI test, as we have the same tests in UI

**Pull request merged**